spec/script/payments_fixer_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "./script/payments_fixer"
+
+RSpec.describe PaymentsFixer do
+  subject(:fix) { described_class.fix_all! }
+
+  before do
+    create_list(:pfmp, 2, :validated)
+  end
+
+  it "goes well" do
+    expect { fix }.not_to raise_error
+  end
+
+  context "when there is an exta payment for a pfmp" do
+    before do
+      create(:payment, pfmp: Pfmp.last)
+    end
+
+    it "deletes the extra payments" do
+      expect { fix }.to change(Payment, :count).by(-1)
+    end
```

Comment:
```suggestion
    let!(:extra_payment) do
      create(:payment, pfmp: Pfmp.last)
    end

    it "deletes the extra payments" do
      expect { fix }.to change(Payment, :count).by(-1)
      expect { extra_payment.reload }.to raise_error(ActiveRecord::RecordNotFound)
    end
```

Reply:
Le linter aime pas qu'il y ait 2 `expect` dans le même `it`

Reply:
Mais je vais garder le test du raise error, c'est mieux.

Reply:
Ouais ça m'étonne pas, j'ai écris les 2 formes pour que tu comprennes mon intention, on peut faire mieux.